### DISCUSSION TITLE
Error OS return value -11

### DISCUSSION THREAD

**Discussion by CorentinEscobar** [ablab/spades#1222] [Q&A]
Hi everyone,

I have this message error 
== Error ==  system call for: "['/home/horigene/Unicycler/SPAdes-3.15.4-Linux/bin/spades-core', '/home/horigene/Unicycler/test_illumina/spades_assembly/K21/configs/config.info', '/home/horigene/Unicycler/test_illumina/spades_assembly/K21/configs/isolate_mode.info']" finished abnormally, OS return value: -11

I didn't find how to fix it. Do you have any idea ?
Thanks

**Comment by asl** [comment]
Please open an issue attaching the relevant files as instructed
